src/service/VerifyService.ts

```diff
@@ -161,40 +175,76 @@ export class VerifyService {
         }
         return { header, message: `Your are not the sudo user!` };
     }
+    shouldRun(): Promise<boolean> {
+        return Promise.resolve(!BootstrapUtils.isWindows());
+    }
+}
 
-    public async loadVersionFromCommand(command: string): Promise<string | undefined> {
-        return this.loadVersion((await BootstrapUtils.exec(this.logger, command)).stdout.trim());
+export class VerifyService {
+    private readonly expectedVersions: ExpectedVersions;
+
+    public actions: VerifyAction[] = [];
+
+    constructor(private readonly logger: Logger, expectedVersions: Partial<ExpectedVersions> = {}) {
```

Comment:
why is this a Partial. when would you want to override default requirements?

Reply:
For unit testing